com.microsoft.mrtk.input/Subsystems/SyntheticHands/SyntheticHandsSubsystem.cs

```diff
@@ -52,28 +53,89 @@ private static void Register()
         /// <returns>
         /// Identifier representing the hand pose.
         /// </returns>
-        public GestureId GetNeutralPose(XRNode handNode)
-        {
-            return (provider as SynthesisProvider).GetNeutralPose(handNode);
-        }
-
+        [Obsolete("Please use the GetNeutralHandshape(handNode) instead.")]
+        public HandshapeId GetNeutralPose(XRNode handNode) => GetNeutralHandshape(handNode);
         /// <summary>
         /// Sets the neutral pose for the specified hand.
         /// </summary>
         /// <param name="handNode">The hand for which the neutral pose is being set.</param>
         /// <param name="poseId">The desired hand pose.</param>
-        public void SetNeutralPose(XRNode handNode, GestureId poseId)
+        [Obsolete("Please use the SetNeutralHandshape(handNode, handshapeId) instead.")]
+        public void SetNeutralPose(XRNode handNode, HandshapeId poseId) => SetNeutralHandshape(handNode, poseId);
+
+        /// <summary>
+        /// Requests the selection pose for the specified hand.
+        /// </summary>
+        /// <param name="handNode">The hand for which the selection pose is being requested.</param>
+        /// <returns>
+        /// Identifier representing the hand pose.
+        /// </returns>
+        [Obsolete("Please use the GetSelectionHandshape(handNode) instead.")]
+        public HandshapeId GetSelectionPose(XRNode handNode) => GetSelectionHandshape(handNode);
+
+        /// <summary>
+        /// Sets the selection pose for the specified hand.
+        /// </summary>
+        /// <param name="handNode">The hand for which the selection pose is being set.</param>
+        /// <param name="poseId">The desired hand pose.</param>
+        [Obsolete("Please use the SetSelectionHandshape(handNode, handshapeId) instead.")]
+        public void SetSelectionPose(XRNode handNode, HandshapeId poseId) => SetSelectionHandshape(handNode, poseId);
+
+        /// <summary>
+        /// Requests the neutral handshape for the specified hand.
+        /// </summary>
+        /// <param name="handNode">The hand for which the neutral handshape is being requested.</param>
+        /// <returns>
+        /// Identifier representing the hand handshape.
+        /// </returns>
+        public HandshapeId GetNeutralHandshape(XRNode handNode)
+        {
+            return (provider as SynthesisProvider).GetNeutralHandshape(handNode);
+        }
+
+        /// <summary>
+        /// Sets the neutral handshape for the specified hand.
+        /// </summary>
+        /// <param name="handNode">The hand for which the neutral handshape is being set.</param>
+        /// <param name="handshapeId">The desired hand handshape.</param>
+        public void SetNeutralHandshape(XRNode handNode, HandshapeId handshapeId)
         {
-            (provider as SynthesisProvider).SetNeutralPose(handNode, poseId);
+            (provider as SynthesisProvider).SetNeutralHandshape(handNode, handshapeId);
```

Comment:
Should we just cache the provider as a `SynthesisProvider` somewhere, so we're not constantly casting?

Reply:
Just noticed this isn't actually part of your change, so we could optimize later.

Reply:
Sounds like a good idea, will do